cabal.project

```diff
@@ -9,7 +9,7 @@
 
 -- `import:` is only available to users of Cabal (the tool) >= 3.8.1.0.
 
-import: https://www.stackage.org/nightly-2022-09-05/cabal.config
+import: stackage/ghc-9.2.4/cabal.config
```

Comment:
Now that there is no connection between the `cabal.config` file and a Stackage snapshot, I suggest `cabal.config` is moved the root directory of the repository, to sit beside the `stack.yaml`.

The Haskell comment immediately above is no longer apt - perhaps it should be replaced by a reference to the new `stack ls dependencies` subcommand or the script that makes use of it?

Reply:
I'd like to leave the comment there in the project about the limitations as stack-only users can be quite surprised by this. I've added a comment mentioning that `ls dependencies constraints` supplies the `cabal.config` constraints.